dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5165,7 +5165,7 @@ public String toString() {
           .build();
   public static final PropertyKey USER_FILE_REPLICATION_MIN =
       intBuilder(Name.USER_FILE_REPLICATION_MIN)
-          .setDefaultValue(0)
+          .setDefaultValue(1)
```

Comment:
the default value should stay `0` as a minimum of 1 replica means the file always has to be cached on at least one worker. This prevents the pages of the file from being evicted.

Reply:
OK

Reply:
@dbw9580 Please take another look.